contracts/volt/IVolt.sol

```diff
@@ -38,3 +38,7 @@ interface IVolt is IERC20 {
 
     function mint(address account, uint256 amount) external;
 }
+
+interface IVoltBurn is IVolt {
+    function burnFrom(address account, uint256 amount) external;
+}
```

Comment:
why not add this in IVolt directly ? IVolt has burn(amount)

Reply:
adding it into Volt means you have to modify the implementation because multiple places define this burnFrom function, which means you have to override it in the base Volt contract.